src/render.cc

```diff
@@ -47,9 +47,10 @@ AbstractNode *RenderModule::instantiate(const Context *ctx, const ModuleInstanti
 	auto node = new RenderNode(inst);
 
 	AssignmentList args{Assignment("convexity")};
+	AssignmentList optargs;
 
 	Context c(ctx);
-	c.setVariables(args, evalctx);
+	c.setVariables(evalctx, args, optargs);
```

Comment:
In other places, you're just relying on the default value of optargs. Why not here?

Reply:
This is just an artifact from debugging/cleanup.